lib/src/main/kotlin/com/swmansion/starknet/crypto/StarknetCurve.kt

```diff
@@ -15,7 +16,7 @@ import java.math.BigInteger
  * @return small endian byte array
  */
 private fun bigintToNative(input: BigInteger): ByteArray {
-    val converted = input.toByteArray().apply { reverse() }
+    val converted = input.toBytes().apply { reverse() }
```

Comment:
`toBytes` is more correct here, although it won't make a difference since bigintegers here come from Felt which can't use 256 bits. It is better to improve it in case someone uses this function for something else, without the additional constraints.